src/Service/EmailService.php

```diff
@@ -0,0 +1,137 @@
+<?php
+namespace App\Service;
+
+use App\Entity\User;
+use Aws\Ses\SesClient;
+use Aws\Exception\AwsException;
+use Doctrine\Common\Persistence\ObjectManager;
+use mysql_xdevapi\Exception;
+use phpDocumentor\Reflection\Types\Boolean;
+
+class EmailService {
+
+    /**
+     * @var \Swift_Mailer
+     *
+     */
+    private $mailer;
+    private $templating;
```

Comment:
phpdoc blocs on the other vars as well ;)

Reply:
I let you check the last commit that i've done.

Reply:
k, talk about this later